rdtools/normalization.py

```diff
@@ -381,3 +381,37 @@ def check_series_frequency(series, series_description):
     else:
         freq = series.index.freq
     return freq
+
+
+def energy_from_power(power_series, max_interval_hours):
+    '''
+    Returns a right-labeled energy time series from an instantaneous power time series.
+    Energy is not returned when the gap between data points exceeds max_interval_hours.
+    power_series should be given in Watts, and max_interval_hours in hours.
+
+    Parameters
+    ----------
+    power_series: Pandas Series with DatetimeIndex
+        Instantaneous time series of PV power generation in Watts
+    max_interval_hours: float
+        The maximum allowed gap between power measurements. If the gap between
+        consecutive power measurements exceeds max_interval_hours, no energy value
+        will be returned for that interval
+
+    Returns:
+    --------
+    right-labeled energy pandas time series in Wh
+
+    '''
+
+    if not isinstance(power_series.index, pd.DatetimeIndex):
+        raise ValueError('power_series must be a pandas series with a DatetimeIndex')
+
+    times = np.array(power_series.index.astype('int64'))
+    time_deltas = np.diff(times) / 10.0**9 / 3600.0
+
```

Comment:
```suggestion
if max_interval_hours is None:
    max_interval_hours = np.median(time_deltas) * 4
```

Reply:
PS:   `max_interval_hours = None`  also needs to be added to the function input if you do this.

Reply:
Ok, sure - we can set the default gap at the median timestep.  I'll approve this.

Reply:
`energy_from_power()` can be called with a single argument, resolving this suggestion.